aptos-move/framework/aptos-framework/doc/aggregator_v2.md

```diff
@@ -82,23 +68,10 @@ read, read_snapshot, read_derived_string
     -  [Function `derive_string_concat`](#@Specification_1_derive_string_concat)
     -  [Function `copy_snapshot`](#@Specification_1_copy_snapshot)
     -  [Function `string_concat`](#@Specification_1_string_concat)
-    -  [Function `verify_aggregator_try_add_sub`](#@Specification_1_verify_aggregator_try_add_sub)
```

Comment:
Do we have a bug open for doc generator, where there is a change in the behavior, when using v2, verify_only functions are skipped? Or perhaps this is intended behavior?

Reply:
Good catch! It is not intended. Filed a bug here: https://github.com/aptos-labs/aptos-core/issues/15687